### ISSUE TITLE
Warnings in the TTS Console for `gs.down.send` event

### ISSUE THREAD

**Issue by i3mSuperMan** [TheThingsNetwork/lorawan-stack#4078]
#### Summary
We are seeing a warning, i.e., `There was an error and events cannot be displayed and The raw data can be viewed by clicking this row` in the gateway live data of `TTSC` Console for the `gs.down.send` event.
#### Steps to Reproduce

1. Register an end-device and Gateway in TTSC Console.
2. Schedule a `downlink`.
3. You could see the `successfully schedule data downlink` event in the end-device i.e., `ns.down.data.schedule.success`
4. Navigate to the gateway live data, observe `gs.down.send` event is shown as a warning i.e., `There was an error and events cannot be displayed. The raw data can be viewed by clicking this row`

#### What do you see now?

- The warning is seen whenever a `downlink` is scheduled from the Network Server, but we can see the raw event by clicking on it (refer to the image).
- Observed across all TTSC clusters.
![image](https://user-images.githubusercontent.com/65446107/115502688-effa1100-a292-11eb-982d-7c5432dde529.png)
![image](https://user-images.githubusercontent.com/65446107/115502730-030ce100-a293-11eb-8dae-a51205bb32c2.png)
![image](https://user-images.githubusercontent.com/65446107/115502752-0a33ef00-a293-11eb-8137-24fc163569d9.png)

#### What do you want to see instead?
 `gs.down.send` event.
#### Environment
TTS v3.12.1
#### How do you propose to implement this?
Not sure
#### How do you propose to test this?
...
#### Can you do this yourself and submit a Pull Request?
No.

**Comment by KrishnaIyer** [comment]
assigning @kschiffer  now so that he can assess/reassign.

**Comment by johanstokking** [comment]
@i3mSuperMan what does the CLI report?

@KrishnaIyer this is very likely related to the `downlink_sent_total` metric not being incremented either.

**Comment by i3mSuperMan** [comment]
@johanstokking 
The CLI events doesn’t seem to report the warning. They include `gs.down.send` event and everything seems fine.

**Comment by mauimauer** [comment]
I am seeing the same (for downlink ACKs)

**Comment by kschiffer** [comment]
This is a display logic issue in the Console. I'm working on a fix.

@i3mSuperMan can you post an example event payload (as text)?

**Comment by i3mSuperMan** [comment]
@kschiffer, Here is an example of the `gs.down.send` event payload for the confirmed downlink
```
{
  "name": "gs.down.send",
  "time": "2021-04-29T10:14:05.406196093Z",
  "identifiers": [
    {
      "gateway_ids": {
        "gateway_id": "",
        "eui": ""
      }
    }
  ],
  "data": {
    "@type": "type.googleapis.com/ttn.lorawan.v3.DownlinkMessage",
    "raw_payload": "oGJQCCaAcgUBIGSB4tU=",
    "request": {
      "downlink_paths": [
        {
          "uplink_token": "ChsKGQoNdGVrdGVsaWMta29uYRIIZH/a//4Ae9QQ/OH4swwaDAjrjaqEBhCPxrzRAyDgsL3n9a0R"
        }
      ],
      "rx1_delay": 5,
      "rx1_data_rate_index": 5,
      "rx1_frequency": "868100000",
      "rx2_data_rate_index": 3,
      "rx2_frequency": "869525000",
      "frequency_plan_id": "EU_863_870_TTN"
    },
    "correlation_ids": [
      "as:downlink:01F4EHY37VFHS2NH33XWQZ7K8T",
      "gs:conn:01F4C94B13YZRGP62GET7TP2GR",
      "gs:up:host:01F4C94B1BM14V4Y6MER7VA470",
      "gs:uplink:01F4EHY2E8PRMKEQWCB2DV96M5",
      "ns:downlink:01F4EHY3TWB2HBDQ6YBMFV2TZ6",
      "ns:uplink:01F4EHY2F2SQ93K4KPGEHBQT7T",
      "rpc:/ttn.lorawan.v3.AppAs/DownlinkQueueReplace:5f808fba-45f1-44c9-a496-b7d3a516c20e",
      "rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01F4EHY2F2EQ9F0RKDWHSCK5RG",
      "gs:conn:01F4C94B13YZRGP62GET7TP2GR",
      "rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01F4EHY3TYKZV59V52SJEMVQMW"
    ]
  },
  "correlation_ids": [
    "gs:conn:01F4C94B13YZRGP62GET7TP2GR",
    "rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01F4EHY3TYKZV59V52SJEMVQMW"
  ],
  "origin": "ip-10-23-13-196.eu-west-1.compute.internal",
  "context": {
    "tenant-id": ""
  },
  "visibility": {
    "rights": [
      "RIGHT_GATEWAY_TRAFFIC_READ"
    ]
  },
  "unique_id": "01F4EHY3TYN2ESW7P5TKNFQ7T7"
} 
```

**Comment by kschiffer** [comment]
Thank you. I'll file a PR that will fix this issue.

> I am seeing the same (for downlink ACKs)

Could you also post an example payload of such event? @mauimauer

If you do, make sure to redact sensitive data.

**Comment by kschiffer** [comment]
Sorry, didn't mean to close this with #4101. I've merged a fix though for the original issue in the OP. Still waiting for more details about downlink ack messages.

**Comment by mauimauer** [comment]
Here's where I get the warning
```
{
  "name": "gs.down.send",
  "time": "2021-04-29T13:51:59.289399292Z",
  "identifiers": [
    {
      "gateway_ids": {
        "gateway_id": "redacted",
        "eui": "redacted"
      }
    }
  ],
  "data": {
    "@type": "type.googleapis.com/ttn.lorawan.v3.DownlinkMessage",
    "raw_payload": "YPpvCyaljgUDUv8AAfxVtm4=",
    "request": {
      "downlink_paths": [
        {
          "uplink_token": "Ch0KGwoPbWF6LXVrLWxkbi1hZzAxEgjcpjL//p+CbxDsi42SDBoMCP7zqoQGEOXi5qEDIODDpLbtvQs="
        }
      ],
      "rx1_delay": 5,
      "rx1_frequency": "867100000",
      "rx2_data_rate_index": 3,
      "rx2_frequency": "869525000",
      "priority": "HIGHEST",
      "frequency_plan_id": "EU_863_870_TTN"
    },
    "correlation_ids": [
      "gs:conn:01F4DE7T56CEFZTGY7H1AH4J7G",
      "gs:up:host:01F4DE7T5FK29DSTNXNJ7NMYF3",
      "gs:uplink:01F4EYD2WW77AX1CBX27ANCAND",
      "ns:downlink:01F4EYD39RSWFAP0B11K0ESWXY",
      "ns:uplink:01F4EYD2WXS46AN7BYYVQACWQQ",
      "rpc:/ttn.lorawan.v3.GsNs/HandleUplink:01F4EYD2WXJZ1AZCJ0XBJD1ZHX",
      "gs:conn:01F4DE7T56CEFZTGY7H1AH4J7G",
      "rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01F4EYD39SVY6P7BRV7D3GR2NC"
    ]
  },
  "correlation_ids": [
    "gs:conn:01F4DE7T56CEFZTGY7H1AH4J7G",
    "rpc:/ttn.lorawan.v3.NsGs/ScheduleDownlink:01F4EYD39SVY6P7BRV7D3GR2NC"
  ],
  "origin": "ip-10-100-12-202.eu-west-1.compute.internal",
  "context": {
    "tenant-id": "redacted"
  },
  "visibility": {
    "rights": [
      "RIGHT_GATEWAY_TRAFFIC_READ"
    ]
  },
  "unique_id": "01F4EYD39SWT4HBWQNGEKVRGV2"
}
```

**Comment by kschiffer** [comment]
Thank you very much @mauimauer.

I can confirm that my PR solves the display issue with this event as well. Note though that this is not a downlink ack event, which have the `name`: `gs.down.tx.success`.

The merged fix will be part of the `3.12.2` release which is scheduled for this week and to be deployed shortly thereafter.
